<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerGUI</class>
 <widget class="QMainWindow" name="MixerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mixer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Default Tray Device:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_default"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>153</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose_Mixer"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Mixer_and_Tray"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_Mixer_and_Tray">
   <property name="text">
    <string>Close Mixer and Tray</string>
   </property>
  </action>
  <action name="actionClose_Mixer">
   <property name="text">
    <string>Hide Mixer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
